src/components/ReduxFormCheckbox/index.jsx

```diff
@@ -28,6 +30,7 @@ const ReduxFormCheckbox = (props) => {
 
 ReduxFormCheckbox.defaultProps = {
   helptext: null,
+  defaultChecked: false,
```

Comment:
shouldn't this be `true` as well?

Reply:
No this component is being used in Revoke modal as well which requires the default state to be unchecked.